incubator/hnc/pkg/controllers/object_controller.go

```diff
@@ -104,6 +105,12 @@ func (r *ObjectReconciler) SyncNamespace(ctx context.Context, log logr.Logger, n
 // update deletes this object if it's an obsolete copy, and otherwise ensures it's been propagated
 // to any child namespaces.
 func (r *ObjectReconciler) update(ctx context.Context, log logr.Logger, inst *unstructured.Unstructured) error {
+	//Checks to see if the object is correct and should propagate
+	if canPropagate := r.ensureCanPropagate(inst.GetNamespace(), inst); !canPropagate {
```

Comment:
No need to introduce a new variable here:

`if !r.ensureCanPropagate(inst.GetNamespace(), inst) {`